functions/Rename-DbaLogin.ps1

```diff
@@ -133,7 +133,7 @@ function Rename-DbaLogin {
 
             if ($Force) {
                 foreach ($db in $dbenums) {
-                    $db = $databases[$db.DBName]
+                    $db = $databases | Where-Object Name -eq $db.DBName
```

Comment:
I don't like that `$db` is used for two different things here. As `$dbenums` is set as `$dbenums = $currentLogin.EnumDatabaseMappings()` in line 104, the foreach in line 135 should use something like `$mapping in $dbenums` or `$dbMapping in $dbenums`. One step further would be to rename `$dbenums` because the name of the variable has no hint about what kind of enumeration is inside the variable.

But the change in line 136 fixes the problem, sou this could be merged...

Reply:
i agree, it was confusing to me too. i was like... there's no dbName in db. gonna merge and can fix the naming later as i have barely enough umph to do this one.

Reply:
Opened #8340 for that.